[2024-07-21 01:38:46,155: INFO: main: Welcome to our custom log]
[2024-07-22 20:33:30,647: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-22 20:33:30,652: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-22 20:33:30,655: INFO: common: created directory at: artifacts]
[2024-07-22 20:33:30,657: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-22 20:33:30,658: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-22 20:34:21,873: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-22 20:36:45,884: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-22 20:36:45,888: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-22 20:36:45,890: INFO: common: created directory at: artifacts]
[2024-07-22 20:36:45,892: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-22 20:36:45,894: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-22 20:46:10,779: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-23 00:23:21,032: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-23 00:23:21,032: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-23 00:23:21,033: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-23 00:23:21,033: INFO: common: created directory at: artifacts]
[2024-07-23 00:23:21,033: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-23 00:23:21,033: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-23 00:29:45,275: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-23 00:29:45,276: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-23 00:29:45,277: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-23 00:29:45,277: INFO: common: created directory at: artifacts]
[2024-07-23 00:29:45,277: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-23 00:29:45,277: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-23 00:30:04,776: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-23 00:30:04,777: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-23 00:30:04,777: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-23 00:30:04,777: INFO: common: created directory at: artifacts]
[2024-07-23 00:30:04,777: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-23 00:30:04,777: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-23 00:41:13,340: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-23 00:41:18,120: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-23 01:03:12,390: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-23 01:03:12,395: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-23 01:03:12,397: INFO: common: created directory at: artifacts]
[2024-07-23 01:03:12,398: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-23 01:04:18,148: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-23 01:04:18,153: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-23 01:04:18,155: INFO: common: created directory at: artifacts]
[2024-07-23 01:04:18,156: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-23 01:04:18,703: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-23 01:21:28,960: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-23 01:21:28,961: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-23 01:21:28,962: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-23 01:21:28,962: INFO: common: created directory at: artifacts]
[2024-07-23 01:21:28,962: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-23 01:21:28,962: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-23 01:32:28,756: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-23 01:32:34,512: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-23 01:32:34,512: INFO: main: *******************]
[2024-07-23 01:32:34,512: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-23 01:32:34,513: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-23 01:32:34,514: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-23 01:32:34,514: INFO: common: created directory at: artifacts]
[2024-07-23 01:32:34,543: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-23 01:32:35,022: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-23 01:32:35,190: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-24 19:36:41,344: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 19:36:41,367: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 19:36:41,369: INFO: common: created directory at: artifacts]
[2024-07-24 19:36:41,371: INFO: common: created directory at: artifacts/training]
[2024-07-24 19:37:53,280: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 19:37:53,285: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 19:37:53,287: INFO: common: created directory at: artifacts]
[2024-07-24 19:37:53,289: INFO: common: created directory at: artifacts/training]
[2024-07-24 19:53:03,797: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-24 19:53:03,798: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 19:53:03,799: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 19:53:03,799: INFO: common: created directory at: artifacts]
[2024-07-24 19:53:03,799: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-24 19:53:03,799: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-24 20:00:38,936: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-24 20:00:43,410: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-24 20:00:43,411: INFO: main: *******************]
[2024-07-24 20:00:43,411: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-24 20:00:43,412: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 20:00:43,412: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 20:00:43,412: INFO: common: created directory at: artifacts]
[2024-07-24 20:00:43,412: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-24 20:00:43,829: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-24 20:00:43,952: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-24 20:00:43,953: INFO: main: *******************]
[2024-07-24 20:00:43,953: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-24 20:00:43,953: ERROR: main: name 'ModelTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "main.py", line 34, in <module>
    model_trainer = ModelTrainingPipeline()
NameError: name 'ModelTrainingPipeline' is not defined
[2024-07-24 20:05:30,463: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-24 20:05:30,464: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 20:05:30,465: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 20:05:30,465: INFO: common: created directory at: artifacts]
[2024-07-24 20:05:30,465: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-24 20:05:30,465: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-24 20:14:27,722: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-24 20:14:32,316: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-24 20:14:32,316: INFO: main: *******************]
[2024-07-24 20:14:32,316: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-24 20:14:32,317: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 20:14:32,318: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 20:14:32,318: INFO: common: created directory at: artifacts]
[2024-07-24 20:14:32,318: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-24 20:14:32,680: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-24 20:14:32,811: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-24 20:14:32,811: INFO: main: *******************]
[2024-07-24 20:14:32,811: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-24 20:14:32,812: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 20:14:32,813: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 20:14:32,813: INFO: common: created directory at: artifacts]
[2024-07-24 20:14:32,813: INFO: common: created directory at: artifacts/training]
[2024-07-24 20:14:33,084: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-ct-scan-image']
Traceback (most recent call last):
  File "main.py", line 36, in <module>
    model_trainer.main()
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/pipeline/stage_03_model_training.py", line 20, in main
    training.train_valid_generator()
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/components/model_training.py", line 37, in train_valid_generator
    self.valid_generator = valid_datagenerator.flow_from_directory(
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 1648, in flow_from_directory
    return DirectoryIterator(
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/keras/preprocessing/image.py", line 563, in __init__
    for subdir in sorted(os.listdir(directory)):
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/kidney-ct-scan-image'
[2024-07-24 20:15:23,430: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-24 20:15:23,431: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 20:15:23,431: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 20:15:23,431: INFO: common: created directory at: artifacts]
[2024-07-24 20:15:23,432: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-24 20:15:23,432: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-24 20:33:53,124: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-24 20:33:53,126: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 20:33:53,126: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 20:33:53,127: INFO: common: created directory at: artifacts]
[2024-07-24 20:33:53,127: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-24 20:33:53,127: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-24 20:42:22,952: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-24 20:42:22,953: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-24 20:42:22,954: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-24 20:42:22,954: INFO: common: created directory at: artifacts]
[2024-07-24 20:42:22,954: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-24 20:42:22,954: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 18:06:18,929: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-25 18:06:18,932: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 18:06:18,934: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 18:06:18,935: INFO: common: created directory at: artifacts]
[2024-07-25 18:06:18,935: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-25 18:06:18,935: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 18:06:18,940: ERROR: main: HTTPSConnectionPool(host='drive.google.com', port=443): Max retries exceeded with url: /uc?/export=download&id=1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fae962c73d0>: Failed to resolve 'drive.google.com' ([Errno -3] Temporary failure in name resolution)"))]
Traceback (most recent call last):
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connection.py", line 196, in _new_conn
    sock = connection.create_connection(
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/util/connection.py", line 60, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/socket.py", line 918, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connectionpool.py", line 789, in urlopen
    response = self._make_request(
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connectionpool.py", line 490, in _make_request
    raise new_e
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connectionpool.py", line 466, in _make_request
    self._validate_conn(conn)
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connectionpool.py", line 1095, in _validate_conn
    conn.connect()
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connection.py", line 615, in connect
    self.sock = sock = self._new_conn()
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connection.py", line 203, in _new_conn
    raise NameResolutionError(self.host, self, e) from e
urllib3.exceptions.NameResolutionError: <urllib3.connection.HTTPSConnection object at 0x7fae962c73d0>: Failed to resolve 'drive.google.com' ([Errno -3] Temporary failure in name resolution)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/requests/adapters.py", line 667, in send
    resp = conn.urlopen(
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/connectionpool.py", line 843, in urlopen
    retries = retries.increment(
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/urllib3/util/retry.py", line 519, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='drive.google.com', port=443): Max retries exceeded with url: /uc?/export=download&id=1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fae962c73d0>: Failed to resolve 'drive.google.com' ([Errno -3] Temporary failure in name resolution)"))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/pipeline/stage_01_data_ingestion.py", line 16, in main
    data_ingestion.download_file()
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/components/data_ingestion.py", line 31, in download_file
    raise e
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/components/data_ingestion.py", line 26, in download_file
    gdown.download(prefix+file_id,zip_download_dir)
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/gdown/download.py", line 202, in download
    res = sess.get(url, stream=True, verify=verify)
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/requests/sessions.py", line 602, in get
    return self.request("GET", url, **kwargs)
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/requests/adapters.py", line 700, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='drive.google.com', port=443): Max retries exceeded with url: /uc?/export=download&id=1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x7fae962c73d0>: Failed to resolve 'drive.google.com' ([Errno -3] Temporary failure in name resolution)"))
[2024-07-25 18:10:44,384: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-25 18:10:44,385: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 18:10:44,386: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 18:10:44,386: INFO: common: created directory at: artifacts]
[2024-07-25 18:10:44,386: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-25 18:10:44,386: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 18:30:22,588: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-25 18:30:22,589: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 18:30:22,590: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 18:30:22,590: INFO: common: created directory at: artifacts]
[2024-07-25 18:30:22,590: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-25 18:30:22,590: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 18:58:40,623: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-25 18:58:40,624: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 18:58:40,625: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 18:58:40,625: INFO: common: created directory at: artifacts]
[2024-07-25 18:58:40,625: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-25 18:58:40,625: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 18:58:46,819: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    data_ingestion.main()
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/pipeline/stage_01_data_ingestion.py", line 16, in main
    data_ingestion.download_file()
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/components/data_ingestion.py", line 31, in download_file
    raise e
  File "/home/ketaki/speProject/Kidney-Disease-Classification-MLFlow-DVC/src/cnnClassifier/components/data_ingestion.py", line 26, in download_file
    gdown.download(prefix+file_id,zip_download_dir)
  File "/home/ketaki/anaconda3/envs/kidney/lib/python3.8/site-packages/gdown/download.py", line 304, in download
    for file in os.listdir(osp.dirname(output) or "."):
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion'
[2024-07-25 18:58:57,966: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-25 18:58:57,967: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 18:58:57,968: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 18:58:57,969: INFO: common: created directory at: artifacts]
[2024-07-25 18:58:57,969: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-25 18:58:57,970: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1kwE9Ml5moR1Lo_nYsEUjEimeqYCoyQCV/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 19:03:42,356: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-25 19:03:42,358: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 19:03:42,359: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 19:03:42,359: INFO: common: created directory at: artifacts]
[2024-07-25 19:03:42,359: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-25 19:03:42,359: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 19:04:09,882: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-25 19:04:09,883: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 19:04:09,884: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 19:04:09,884: INFO: common: created directory at: artifacts]
[2024-07-25 19:04:09,884: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-25 19:04:09,884: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 19:07:26,747: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-25 19:07:27,394: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-25 19:07:27,395: INFO: main: *******************]
[2024-07-25 19:07:27,395: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-25 19:07:27,395: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 19:07:27,396: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 19:07:27,396: INFO: common: created directory at: artifacts]
[2024-07-25 19:07:27,396: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-25 19:07:27,849: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-25 19:07:28,037: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-25 19:07:28,037: INFO: main: *******************]
[2024-07-25 19:07:28,037: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-25 19:07:28,039: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-25 19:07:28,040: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-25 19:07:28,040: INFO: common: created directory at: artifacts]
[2024-07-25 19:07:28,040: INFO: common: created directory at: artifacts/training]
[2024-07-25 19:08:55,324: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-07-26 00:08:32,435: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 00:08:32,441: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 00:08:32,443: INFO: common: created directory at: artifacts]
[2024-07-26 00:09:30,503: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 00:09:30,507: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 00:09:30,509: INFO: common: created directory at: artifacts]
[2024-07-26 00:10:06,209: INFO: common: json file saved at: scores.json]
[2024-07-26 00:10:09,723: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-07-26 00:10:09,995: INFO: builder_impl: Assets written to: /tmp/tmpd7d99i02/model/data/model/assets]
[2024-07-26 00:15:52,161: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 00:15:52,167: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 00:15:52,169: INFO: common: created directory at: artifacts]
[2024-07-26 00:16:29,752: INFO: common: json file saved at: scores.json]
[2024-07-26 00:16:32,383: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-07-26 00:16:32,696: INFO: builder_impl: Assets written to: /tmp/tmpufoof6lb/model/data/model/assets]
[2024-07-26 00:40:23,525: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-26 00:40:23,526: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 00:40:23,527: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 00:40:23,527: INFO: common: created directory at: artifacts]
[2024-07-26 00:40:23,527: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-26 00:40:23,527: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 00:40:37,328: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-26 00:40:37,329: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 00:40:37,330: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 00:40:37,330: INFO: common: created directory at: artifacts]
[2024-07-26 00:40:37,330: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-26 00:40:37,330: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 00:42:26,721: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 00:42:27,372: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-26 00:42:27,372: INFO: main: *******************]
[2024-07-26 00:42:27,372: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-26 00:42:27,373: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 00:42:27,374: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 00:42:27,374: INFO: common: created directory at: artifacts]
[2024-07-26 00:42:27,374: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-26 00:42:27,757: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-26 00:42:27,961: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-26 00:42:27,961: INFO: main: *******************]
[2024-07-26 00:42:27,961: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-26 00:42:27,963: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 00:42:27,964: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 00:42:27,965: INFO: common: created directory at: artifacts]
[2024-07-26 00:42:27,965: INFO: common: created directory at: artifacts/training]
[2024-07-26 00:45:59,160: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-07-26 00:45:59,160: INFO: main: *******************]
[2024-07-26 00:45:59,160: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-07-26 00:45:59,161: ERROR: main: name 'EvaluationPipeline' is not defined]
Traceback (most recent call last):
  File "main.py", line 47, in <module>
    model_evalution = EvaluationPipeline()
NameError: name 'EvaluationPipeline' is not defined
[2024-07-26 01:46:47,435: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-26 01:46:47,437: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 01:46:47,438: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 01:46:47,438: INFO: common: created directory at: artifacts]
[2024-07-26 01:46:47,438: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-26 01:46:47,438: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 01:48:15,224: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 01:48:15,883: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-26 01:48:15,883: INFO: main: *******************]
[2024-07-26 01:48:15,883: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-26 01:48:15,884: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 01:48:15,885: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 01:48:15,885: INFO: common: created directory at: artifacts]
[2024-07-26 01:48:15,885: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-26 01:48:16,167: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-26 01:48:16,342: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-26 01:48:16,342: INFO: main: *******************]
[2024-07-26 01:48:16,342: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-26 01:48:16,343: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 01:48:16,344: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 01:48:16,344: INFO: common: created directory at: artifacts]
[2024-07-26 01:48:16,344: INFO: common: created directory at: artifacts/training]
[2024-07-26 01:52:00,627: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-07-26 01:52:00,627: INFO: main: *******************]
[2024-07-26 01:52:00,628: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-07-26 01:52:00,630: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 01:52:00,632: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 01:52:00,633: INFO: common: created directory at: artifacts]
[2024-07-26 01:52:37,498: INFO: common: json file saved at: scores.json]
[2024-07-26 01:52:37,499: INFO: common: json file saved at: scores.json]
[2024-07-26 01:52:37,499: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-07-26 01:58:29,103: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-26 01:58:29,104: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 01:58:29,105: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 01:58:29,105: INFO: common: created directory at: artifacts]
[2024-07-26 01:58:29,105: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-26 01:58:29,105: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 02:00:07,995: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 02:00:08,637: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-26 02:00:08,638: INFO: main: *******************]
[2024-07-26 02:00:08,638: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-26 02:00:08,639: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 02:00:08,639: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 02:00:08,639: INFO: common: created directory at: artifacts]
[2024-07-26 02:00:08,639: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-26 02:00:08,989: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-26 02:00:09,153: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-26 02:00:09,153: INFO: main: *******************]
[2024-07-26 02:00:09,153: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-26 02:00:09,154: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 02:00:09,155: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 02:00:09,155: INFO: common: created directory at: artifacts]
[2024-07-26 02:00:09,156: INFO: common: created directory at: artifacts/training]
[2024-07-26 02:03:41,103: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-07-26 02:03:41,103: INFO: main: *******************]
[2024-07-26 02:03:41,103: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-07-26 02:03:41,105: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 02:03:41,106: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 02:03:41,106: INFO: common: created directory at: artifacts]
[2024-07-26 02:04:13,373: INFO: common: json file saved at: scores.json]
[2024-07-26 02:04:13,373: INFO: common: json file saved at: scores.json]
[2024-07-26 02:04:17,670: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-07-26 02:04:17,930: INFO: builder_impl: Assets written to: /tmp/tmp45la_gf3/model/data/model/assets]
[2024-07-26 02:06:52,370: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-07-26 02:07:55,375: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-07-26 02:07:55,376: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 02:07:55,377: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 02:07:55,377: INFO: common: created directory at: artifacts]
[2024-07-26 02:07:55,377: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-26 02:07:55,377: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 02:09:11,074: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1SD19kKtzvbJuvRH27f4ckmSCqgybhFRQ/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-26 02:09:11,723: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-07-26 02:09:11,723: INFO: main: *******************]
[2024-07-26 02:09:11,723: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-07-26 02:09:11,724: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 02:09:11,725: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 02:09:11,725: INFO: common: created directory at: artifacts]
[2024-07-26 02:09:11,725: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-07-26 02:09:12,000: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-07-26 02:09:12,159: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-07-26 02:09:12,160: INFO: main: *******************]
[2024-07-26 02:09:12,160: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-07-26 02:09:12,161: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 02:09:12,161: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 02:09:12,161: INFO: common: created directory at: artifacts]
[2024-07-26 02:09:12,162: INFO: common: created directory at: artifacts/training]
[2024-07-26 02:12:08,194: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-07-26 02:12:08,194: INFO: main: *******************]
[2024-07-26 02:12:08,194: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-07-26 02:12:08,195: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-26 02:12:08,196: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-26 02:12:08,196: INFO: common: created directory at: artifacts]
[2024-07-26 02:12:34,227: INFO: common: json file saved at: scores.json]
[2024-07-26 02:12:34,228: INFO: common: json file saved at: scores.json]
[2024-07-26 02:12:37,634: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-07-26 02:12:37,967: INFO: builder_impl: Assets written to: /tmp/tmp6u63cpn1/model/data/model/assets]
[2024-07-26 02:14:22,507: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
